pkg/securitycontextconstraints/sccmatching/patched_sc_accessors.go

```diff
@@ -0,0 +1,471 @@
+/*
+ * This file is a copy of k8s.io/kubernetes/pkg/securitycontext/accessors.go that
+ * contains the patch from https://github.com/kubernetes/kubernetes/pull/115968
+ * that only appears in kube 1.27.
+ * FIXME: Remove this file when OpenShift users kube 1.27 as its base.
+ */
+
+package sccmatching
+
+import (
+	"reflect"
+
+	api "k8s.io/kubernetes/pkg/apis/core"
+	"k8s.io/kubernetes/pkg/securitycontext"
+)
+
+type SecccompProfileAccessor interface {
+	SeccompProfile() *api.SeccompProfile
+}
+
+type SeccompProfileMutator interface {
+	SetSeccompProfile(*api.SeccompProfile)
+}
+
+type PatchedPodSecurityContextAccessor interface {
+	securitycontext.PodSecurityContextAccessor
+	SecccompProfileAccessor
```

Comment:
Usually I would avoid composition, but it might make the change you are doing clearer.